react-app/src/components/projects/AddProject.tsx

```diff
@@ -281,6 +288,13 @@ const AddProject = () => {
               numeric
```

Comment:
In these numeric fields, I can type in text values. Likely a result of one of the other two comments.

Reply:
I couldn't replicate this, but other change was reverted anyway.